### ISSUE TITLE
[zh-CN/policies/security] Security reporting description not translated

### ISSUE THREAD

**Issue by Lsnsh** [rust-lang/www.rust-lang.org#1959]
### Page(s) Affected

- https://www.rust-lang.org/zh-CN/policies/security

### What needs to be fixed?

[zh-CN/policies/security] Security reporting description not translated.

### Suggested Improvement

Add translation for security reporting description in zh-CN/policies/security.

**Comment by Lsnsh** [comment]
I can understand to a certain extent. Returning to the current issue, as a newcomer to translation, I plan to cross use translation tools and manually polish them before create a PR. Do you think this is acceptable.

**Comment by senekor** [comment]
Without somebody to review the PR, it won't be merged unfortunately.

**Comment by Lsnsh** [comment]
For potential PR, may fall into a dead cycle, if no "break".

**Comment by senekor** [comment]
I don't understand this sentence, can you rephrase it?

**Comment by Lsnsh** [comment]
> I think there are currently no trusted people available to review translations, which is why many translation issues are not addressed at the moment.

What I roughly mean is that for pull request,, no reviewer, more backlog, more issue, similar to the situation you mentioned earlier.

Even so, I will create a PR and try to find somebody to review it.

Because before create issue, i want to direct create PR, but i see 

> **tl;dr: Always file an issue first.**

from `CONTRIBUTING.md`, so there is this issue.

I began to gradually understand the meaning of this sentence.

**Comment by senekor** [comment]
> try to find somebody to review it

It would have to be someone that already has some trust among project members, which might be difficult to find if we don't know someone like that ourselves.

> no reviewer, more backlog, more issue

Yeah, the translations are just gonna remain the way they are right now, imperfections included. It's not optimal, but I don't see anything that can be done about it.

**Comment by Lsnsh** [comment]
Got it, thank you very much for your time.

**Comment by senekor** [comment]
fixed by #2228
